submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="img/favicon.png" />
+    <title>Popup</title>
+  </head>
+  <body>
+    <nav>
+      <ul class="nav-container">
+        <li class="nav-el">
+          <a href="#" class="nav-el__item">Mail</a>
+        </li>
+        <li class="nav-el">
+          <a href="#" class="nav-el__item">Images</a>
+        </li>
+
+        <li class="nav-el">
+          <input type="checkbox" class="input_hidden nav-el__menu-input" id="menu-button-input">
+          <label class="nav-el__item nav-el__menu-button" for="menu-button-input">
+            <img src="img/apps_new.png" alt="Menu icon" class="nav-el__menu-image">
+          </label>
+
+          <div class="apps-window">
+            <input type="checkbox" class="input_hidden apps-window__more-input" id="more-button-input">
+            <label class="apps-window__more-button" for="more-button-input">
+              More
+            </label>
+            
+            <div class="apps-cont">
+              <div class="apps-block">
+                <a href="#" class="app-block__item">
+                  <img src="img/calendar.png" class="app-block__img" alt="Calendar icon">
```

Comment:
Can you please check size of the images?
<img width="825" alt="Screenshot 2020-10-30 at 21 42 34" src="https://user-images.githubusercontent.com/42999342/97750342-444ce100-1af9-11eb-8f64-0904dddf2d40.png">

Reply:
Thank you.
I've fixed it)